Codeinwp/themeisle-companion
# Add placeholders for neve

Testing instructions:
- Enable Pagebuilder module for elementor in obfx dashboard.
- The widget placeholders appear only if the page builder module from obfx dashboard is active, Neve theme is active and Neve Pro is not.
- If the condition above is met, try to edit a page with elementor
- In their editor, you should see neve pro blocked widgets
- If Neve pro is enabled, the widgets should not be there

More details here: https://github.com/Codeinwp/neve-pro-addon/issues/1738
